src/vm/method.cpp

```diff
@@ -5055,6 +5055,37 @@ LPVOID NDirectMethodDesc::FindEntryPoint(HINSTANCE hMod) const
 }
 #endif // CROSSGEN_COMPILE
 
+#if !defined(CROSSGEN_COMPILE)
+//*******************************************************************************
+void NDirectMethodDesc::InitEarlyBoundNDirectTarget()
+{
+    CONTRACTL
+    {
+        THROWS;
+        GC_TRIGGERS;
+        INJECT_FAULT(COMPlusThrowOM(););
+    }
+        CONTRACTL_END
```

Comment:
Can we align this and the assert below please.

Reply:
Done.